Source/Core/Common/bbadebug.cpp

```diff
@@ -0,0 +1,20 @@
+// Copyright 2019 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "bbadebug.h"
```

Comment:
Includes should be made relative to the library they're in. e.g.

```cpp
#include "Common/bbadebug.h"
```

(note that, as stated before, Qt shouldn't be in Common anyways)